script.js

```diff
@@ -16,4 +16,173 @@ function toggleMenu() {
+    image: './icons/portfolio-snapshot-desktop.png',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    See_Live: 'https://dhuruvsingh.github.io/Portfolio-Website-New/',
+    Source: 'https://dhuruvsingh.github.io/Portfolio-Website-New/',
+  },
+  {
+    title: 'Multi-Post Stories Gain+Glory',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    image: './icons/portfolio-snapshot-desktop.png',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    See_Live: 'https://dhuruvsingh.github.io/Portfolio-Website-New/',
+    Source: 'https://dhuruvsingh.github.io/Portfolio-Website-New/',
+  },
+  {
+    title: 'Multi-Post Stories Gain+Glory',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    image: './icons/portfolio-snapshot-desktop.png',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    See_Live: 'https://dhuruvsingh.github.io/Portfolio-Website-New/',
+    Source: 'https://dhuruvsingh.github.io/Portfolio-Website-New/',
+  },
+  {
+    title: 'Multi-Post Stories Gain+Glory',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    image: './icons/portfolio-snapshot-desktop.png',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    See_Live: 'https://dhuruvsingh.github.io/Portfolio-Website-New/',
+    Source: 'https://dhuruvsingh.github.io/Portfolio-Website-New/',
+  },
+];
+
+openModalButtons.forEach((button) => {
```

Comment:
- [ ] Kindly ensure all your popup content fit the page, the Figma design doesn't have a scroll feature, I  noticed it in your desktop design;
<img width="433" alt="image" src="https://user-images.githubusercontent.com/61408860/180063122-f2322536-e739-4ae7-bbca-6f403d743fdc.png">

- [ ] It would also be nice reducing the right margin between your image and the card edge so that the right and the left margin are equal. 
<img width="160" alt="image" src="https://user-images.githubusercontent.com/61408860/180064473-64f56d5a-4244-4b63-9aa7-8fe57d9a5ab7.png">

Reply:
**Done**